src/main/java/com/facebook/bitEncoderDecoder/Stage1.java

```diff
@@ -4,11 +4,6 @@
 
     public String encode(String input) {
         StringBuilder encodeString = new StringBuilder();
```

Comment:
It seems that you are trying to delete some of Stage1 code in this pull. I will try to stop you

Reply:
ok, I don't know how to do it